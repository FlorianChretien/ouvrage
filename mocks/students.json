[
  {
    "username": "epatrois",
    "firstName": "Manu",
    "lastName": "Patrois",
    "id": "2",
    "avatar": "https://pbs.twimg.com/profile_images/781247417819725825/g8Rk4ljG_400x400.jpg"
  },
  {
    "id": "5",
    "username": "mdewaele",
    "firstName": "Maxence",
    "lastName": "De Waele",
    "email": "maxencedewaele@gmail.com",
    "birthdate": "838921463",
    "avatar": "https://info.bestfriendspetcare.com/hubfs/BestFriendSpetcare-Sep2016/images/img_dog_right-col-top.png?t=1521138713469"
  },
  {
    "id": "8",
    "username": "rodrigodelanoche",
    "firstName": "Vincent",
    "lastName": "Rodriguez",
    "email": "vincentrodrigu@hotmail.fr",
    "birthdate": "14/05/1413",
    "avatar": "https://s3.amazonaws.com/mixture-mixed/21975/17188/img/photos/portraits/dogs/golden-retriever.jpg"
  },
  {
    "id": "6",
    "username": "erozis",
    "firstName": "Emilie",
    "lastName": "Rozis",
    "email": "emilie.40@hotmail.com",
    "avatar": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSwMFoeG86Nz19Ve8Cf2FqpdiMd9Q8r6cYS1lgmPKZCxJfvb7kK"
  },
  {
    "firstName": "Hubert",
    "username": "hmoncenis",
    "id": "1",
    "lastName": "Montcenis",
    "avatar": "https://barknbrewwi.com/wp-content/uploads/2017/05/bark-and-brew-dog-img.png"
  },
  {
    "id": "7",
    "username": "vivianeglz",
    "firstName": "Viviane",
    "lastName": "Gonzalez",
    "email": "viviane.gonzalez@outlook.fr",
    "birthdate": "02/06/1995",
    "avatar": "https://img.freepik.com/free-photo/cute-golden-retriever-dog_1204-387.jpg?size=338&ext=jpg"
  },
  {
    "id": "3",
    "username": "antoinesinety",
    "firstName": "Antoine",
    "lastName": "de Sinety",
    "email": "fake@yopmail.fr",
    "birthdate": "08/06/1995",
    "avatar": "https://cdn.petcarerx.com/LPPE/images/breedguidethumbs/Akita-dogs-photo.jpg"
  },
  {
    "id": "4",
    "username": "tgauzin",
    "firstName": "Thomas",
    "lastName": "Gauzin",
    "email": "thomas.gauzin@gmail.com"
  }
]
