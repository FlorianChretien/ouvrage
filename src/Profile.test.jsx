import React from 'react';
import Profile from './Profile';
import renderer from 'react-test-renderer';
import students from '../mocks/students.json';

it('renders correctly', () => {
  const tree = renderer
    .create(<Profile student={students[1]}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
