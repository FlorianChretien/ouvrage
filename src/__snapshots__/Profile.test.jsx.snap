// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="wrap-header"
  >
    <h1
      className="title"
    >
      Maxence
       
      De Waele
    </h1>
    <button
      onClick={[Function]}
    >
      Modifier
    </button>
  </div>
  <div
    className="wrap-content"
  >
    <div
      className="avatar"
    >
      <img
        alt="avatar"
        src="https://info.bestfriendspetcare.com/hubfs/BestFriendSpetcare-Sep2016/images/img_dog_right-col-top.png?t=1521138713469"
      />
    </div>
    <div
      className="content"
    >
      <p>
        Maxence
          
        De Waele
      </p>
      <p>
        mdewaele
      </p>
      <p>
        maxencedewaele@gmail.com
      </p>
      <p>
        2018-05-24
      </p>
    </div>
  </div>
</div>
`;
